pkg/cmd/cli/describe/describer.go

```diff
@@ -86,8 +86,8 @@ func (d *BuildDescriber) DescribeParameters(p buildapi.BuildParameters, out *tab
 			formatString(out, "Ref", p.Source.Git.Ref)
 		}
 	}
-	formatString(out, "Output Image", p.Output.ImageTag)
-	formatString(out, "Output Registry", p.Output.Registry)
+	formatString(out, "Output To", p.Output.To)
+	formatString(out, "Output Spec", p.Output.DockerImageReference)
```

Comment:
didn't look like it did to me:
func formatString(out *tabwriter.Writer, label string, v interface{}) {
	fmt.Fprintf(out, fmt.Sprintf("%s:\t%s\n", label, toString(v)))
}